heaps/min_heap.py

```diff
@@ -19,18 +22,32 @@ def __init__(self):
     def add(self, key, value = None):
```

Comment:
✨ However space complexity is O(log n) because of `heap_up`'s recursive call stack